src/state/transactions/updater.tsx

```diff
@@ -24,7 +26,20 @@ export function toSerializableReceipt(receipt: TransactionReceipt): Serializable
   }
 }
 
+function getElapsedTime(): number | undefined {
+  // Only log this metric for the first swap of a session.
+  const ttsMarks = performance.getEntriesByName('tts-mark', 'mark')
+
+  if (ttsMarks.length > 0) {
```

Comment:
This is not reliable, as the performance buffer will empty over time. Instead, you should use a `once` pattern, where you have a local variable that you set so the logic is only executed once. See https://developer.mozilla.org/en-US/docs/Web/API/Performance/setResourceTimingBufferSize.